### PR TITLE
Refactor multilayer drawing function and add headers to convert module

### PR DESCRIPTION
**Pull request by nwlandry**
This adds the ability for users to specify the colors/sizes of every node/edge in the multilayer drawing function.

### CODE REVIEW

**Comment by nwlandry** [comment]
Happy for any feedback @thomasrobiglio!!

**Comment by thomasrobiglio** [comment]
@nwlandry let me know if this is still a draft or you want me to review it (it looks like the function is ready we just need to twitch the tests)... I tried it very quickly today and it seems very nice and much more coherent with the rest of the drawing functions of XGI than my initial attempt 😄, thank you!

**Comment by nwlandry** [comment]
Hi @thomasrobiglio, yes sorry about this - let me draft it and undraft when I am done updating the docs, tests, etc. I had a great foundation to work from, so thanks :)

File: xgi/drawing/draw_utils.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+from matplotlib.colors import LinearSegmentedColormap, ListedColormap
+from numpy import ndarray
+
+from ..exception import XGIError
+from ..stats import IDStat
+from .layout import barycenter_spring_layout
+
+
+def _draw_init(H, ax, pos):
+
```
[the remark this answers is not in the document]
Comment (nwlandry) [reply xgi/drawing/draw_utils.py:34]: Done! Thanks for the suggestion!

**Comment by nwlandry** [comment]
I added a `plane_color` option with the default as grey. Because I rebased, it looks like I reorganized the drawing module, but that is already on the main branch. Thanks for the review!!

**Pull request merged**